spec/models/admin_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'rails_helper'
 
 RSpec.describe Admin, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  active_record_admin = 'activerecord.errors.models.admin.attributes'
+  let(:admin) { FactoryGirl.build(:admin) }
+
+  it "is valid with everything properly filled" do
+    admin.valid?
+    expect(admin).to be_valid
+  end
+
+  it "is invalid without a password" do
+    admin.password = "  "
+    admin.valid?
+    expect(admin.errors[:password]).to include(I18n.t('password.blank', scope: active_record_admin))
+  end
+
+  it "is invalid with a password too short" do
+    admin.password = "a" * 7
+    admin.valid?
+    expect(admin.errors[:password]).to include(I18n.t('password.too_short', count: 8, scope: active_record_admin))
+  end
+
+  it "is invalid with a password too long" do
+    admin.password = "a" * 129
+    admin.valid?
+    expect(admin.errors[:password]).to include(I18n.t('password.too_long', count: 128, scope: active_record_admin))
+  end
+
+  it "is invalid without an email" do
+    admin.email = "   "
+    admin.valid?
+    expect(admin.errors[:email]).to include(I18n.t('email.blank', scope: active_record_admin))
+  end
+
+  it "is invalid with a too short email" do
+    admin.email = "a"
+    admin.valid?
+    expect(admin.errors[:email]).to include(I18n.t('email.invalid', count: 0, scope: active_record_admin))
+  end
+
+  it "is invalid with a too long email" do
+    admin.email = "a" * 255 + "b"
+    admin.valid?
+    expect(admin.errors[:email]).to include(I18n.t('email.invalid', count: 255, scope: active_record_admin))
```

Comment:
You already merged, 5 minutes ago, have you corrected it? Sorry I didn't saw it as soon as you commented.